citadel-internal-service/src/kernel/mod.rs

```diff
@@ -368,6 +368,7 @@ fn spawn_tick_updater(
                                 cid: implicated_cid,
                                 peer_cid,
                                 status: status_message,
+                                request_id: None,
```

Comment:
We would ideally want to make this the same as the request_id as the initial file transfer request that initiated this.